rulebook.tex

```diff
@@ -2121,63 +2145,71 @@ \subsection{Order Schedule}
 production plan. However, each game will use a new randomized order
 schedule.
 
-\subsection{Scheduled Machine Downtime --- Main Track}
+\subsubsection{Scheduled Machine Downtime --- Main Track}
 There will be exactly two occurrences of scheduled machine downtime for each
 game in the main track according to the rules in \refsec{sec:out-of-order}.
 
-\subsection{Tournament Phases}
+\subsubsection{Overview --- Main Track}
 \label{sec:tournament-phases}
-There will be three stages in the main competition, a round-robin
+There will be three stages in the main track, a (optional) round-robin
 phase for all participating teams, playoffs for the best four teams
 from the round-robin phase, and the finals. The best two teams of the
 playoffs play the grand finale to decide which team will become the
-next Logistics League champion, whereas the other two teams compete in
-the small final for the third place.
+winner of the main track in the \ac{RCLL}, whereas the other two teams compete
+in the small final for the third place.
+
+\paragraph{Scoring Scheme}
```

Comment:
might be kinda obvious, but this should mention the finals explicitly too

Reply:
the final is mentioned right before that